tests/run-regressionTest.sh

```diff
@@ -12,8 +12,9 @@ REL_TOL="$6"
 COMPARE_ECL_COMMAND="$7"
 RST_DECK_COMMAND="$8"
 RESTART_STEP="${9}"
-EXE_NAME="${10}"
-shift 10
+RESTART_SCHED="${10}"
+EXE_NAME="${11}"
+shift 11
```

Comment:
this is getting unmanageable. not caused by this addition alone though, but it is time to introduce getopt usage. i will handle that later. that being said there are other drivers which needs updating too, notably the porv and init ones.